backends/smt2/smtbmc.py

```diff
@@ -326,14 +326,8 @@ def write_vlogtb_trace(steps_start, steps_stop, index):
         print("  reg [4095:0] vcdfile;", file=f)
         print("  reg clock = 0, genclock = 1;", file=f)
 
-        primary_inputs = list()
-        clock_inputs = set()
-
-        for name in smt.modinfo[topmod].inputs:
-            if name in ["clk", "clock", "CLK", "CLOCK"]:
```

Comment:
This list, `clk clock CLK CLOCK` is a feature (some might call it hack :) of smtbmc.py, other driver programs might want to use a different list, or use `smtio.py` in a different way altogether.